Client/mods/deathmatch/logic/CClientIMGManager.cpp

```diff
@@ -0,0 +1,110 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto v1.0
+ *               (Shared logic for modifications)
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        mods/shared_logic/CClientIMGManager.cpp
+ *  PURPOSE:     .img container manager class
+ *
+ *****************************************************************************/
+
+#include "StdInc.h"
+
+CClientIMGManager::CClientIMGManager(CClientManager* pManager)
+{
+    // Init
+    m_bRemoveFromList = true;
```

Comment:
Use member init list.
Or even better, just set the default value in the header.